src/components/UserProfile/BaseUserProfile.tsx

```diff
@@ -30,6 +37,9 @@ export const BaseUserProfile: FC<BaseUserProfileProps> = ({user, pinnedItems, lo
                         <ContributionCalendarWrapper>
                             <h2>Contribution Calendar</h2>
                             <ContributionCalendar login={user.login} calendarData={user.contributionsCollection.contributionCalendar}/>
+                            <Link onClick={handleClick}>
+                                <h3 style={{fontWeight: 'lighter'}}>See all repositories</h3>
```

Comment:
We can replace `h3` with styled component

Reply:
that's unnecessary